www/package.json

```diff
@@ -22,6 +22,7 @@
     "emotion-theming": "^10.0.10",
     "fuse.js": "^3.2.0",
     "gatsby": "^2.11.0",
+    "gatsby-design-tokens": "0.0.1",
```

Comment:
Why pin?

Reply:
```suggestion
    "gatsby-design-tokens": "^0.0.1",
```

Reply:
Spoke to @fk about this. He prefers pinning here to ensure publishing design changes to `www` are deliberate.